basket.js

```diff
@@ -9,9 +9,11 @@ const Basket = {
 
   async subscribe(email) {
     const params = {
+      form: true,
       email,
       newsletters: AppConstants.BASKET_NEWSLETTER,
       validated: true,
```

Comment:
this parameter will be ignored by basket. I've just realized that it's in the docs, but it was removed from the code at some point. I'll update basket's docs, but you can remove it here as well if you want. It won't break anything, to be clear, but it is superfluous at this point.

Reply:
Oh! this is an option to `got`. I think this should go down in the `reqOptions` object.

Reply:
removed.